src/mds/MDSRank.cc

```diff
@@ -934,6 +934,12 @@ void MDSRank::respawn()
   }
 }
 
+void MDSRank::abort(std::string_view msg)
+{
+  monc->flush_log();
+  ceph_abort(msg);
+}
+
```

Comment:
There should be other places too where this might be essential to do right? Would it be good to starting using this in all cases instead of `ceph_abort()`?

Reply:
There may be one or two other paths which this would be useful. I've opened a tracker ticket here: https://tracker.ceph.com/issues/62208